File: java/oop/CircleApp/src/com/techlabs/circle/Circle.java
package com.techlabs.circle;

public class Circle {	
	
	private float radius;
	private BorderOptionType bStyle;
	
	public Circle(float radius,BorderOptionType bStyle) {
		this.radius=radius;
		this.bStyle=bStyle;
	}
	
	public float getRadius(){
		return this.radius;
	}
	
	public BorderOptionType getBorderStyle(){
		return bStyle;
	}
	
	public float area(){
		return (float) (2*Math.PI*radius*radius);
	}
}

File: dotNet/CSharp/CircleApp/CircleApp/Circle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CircleApp
{
    class Circle
    {
        private float _radius;
        private BorderType _borderStyle;

        public Circle(float radius,BorderType borderStyle)
        {
            this._radius = radius;
            this._borderStyle = borderStyle;
        }

        public float Radius
        {
            get
            {
                return _radius;
            }
        }

        public BorderType BorderStyle
        {
            get
            {
                return _borderStyle;
            }
        }

        public float Area()
        {
            return (float)(2 * Math.PI * Radius * Radius);
        }

        public override string ToString()
        {
            return "Area: "+Area()
                +"\nStyle: "+BorderStyle;
        }
    }
}
